### ISSUE TITLE
[BUG] multi-return value functions and error checking using chained .Error().

### ISSUE THREAD

**Issue by thediveo** [nunnatsa/ginkgolinter#228]
**Describe the bug**

ginkgolinter doesn't take a chained `.Error().` in assertions into account. The reproducer below triggers:

```
ginkgo-linter: MatchError first parameter (42) must be error, string, GomegaMatcher or func(error)bool are allowed (ginkgolinter) go-golangci-lint-v2
```

**To Reproduce**

```go
func f() (int, error) { return 42, nil }

var _ = Describe("foo", func() {
	It("bar", func() {
		Expect(f()).Error().To(MatchError(42))
	})
})
```

**Expected behavior**

No ginkgolinter error for the above assertion.

**Environment:**
 - OS: linux devcontainer
 - Go: 1.26.1
 - Version: dep github.com/nunnatsa/ginkgolinter v0.23.0

**Comment by thediveo** [comment]
What the `.Error().` does is to pick the last value of a multi-value actual and from there there's the normal handling; `Error()` basically returns a new assertion that then has only a single actual value.

**Comment by thediveo** [comment]
Please ignore me, I can't read the linting message correctly, so shame on me. Closing as invalid.
